vendor/ocaml/typing/printtyp.ml

```diff
@@ -1457,6 +1479,69 @@ let unification_error unif tr txt1 ppf txt2 =
     with exn ->
       print_labels := true;
       raise exn
+#else
+
+let super_trace ppf =
+  let rec super_trace first_report ppf = function
+    | (t1, t1') :: (t2, t2') :: rem ->
+      fprintf ppf 
+        "@,@,@[<v 2>";
+      if first_report then 
+        fprintf ppf "The incompatible parts:@,"
+      else begin 
+        fprintf ppf "Further expanded:@,"
+      end;
+      fprintf ppf 
+        "@[<v>\
```

Comment:
lol I'm just following the old `unification_error`'s formatting. I'll change it if you want